### PR TITLE
Searchbox/Textfield: Fix high contrast borders when disabled

### LINKED ISSUE #10624: Searchbox/Textfield: No visual difference when disabled in high contrast mode
pulled from #7500

A standard `<input disabled />` is not visually different from an enabled one in high contrast mode (not something we broke in Fabric). Disabled labels look different, but the inputs (borders/background) and place holder text are identical.

image

TextField

image

### PR DESCRIPTION
**Pull request by micahgodbolt**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #10624
- [ ] Include a change request file using `$ yarn change`

Before:

65621535-cbd03400-df78-11e9-9a8b-5a19032b22e5
image

After:

image

image

image

image

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11035)

### CODE REVIEW

**Comment by ecraig12345** [comment]
I think we should either integrate this into @betrue-final-final's PR #10885 or at least wait until that merges, to avoid making Ben deal with even more conflicts.

**Comment by dzearing** [comment]
@micahgodbolt the above PR was merged. Can you rebase this and get it merged? Thank you!

**Comment by micahgodbolt** [comment]
All set

File: packages/office-ui-fabric-react/src/components/TextField/TextField.styles.tsx [hunk 1]
```diff
@@ -185,16 +190,7 @@ export function getStyles(props: ITextFieldStyleProps): ITextFieldStyles {
...
-        selectors: {
-          ':hover': {
-            selectors: {
-              [HighContrastSelector]: {
-                borderColor: 'Highlight'
-              }
-            }
-          }
-        }
+        position: 'relative'
```
Comment (ecraig12345) [inline packages/office-ui-fabric-react/src/components/TextField/TextField.styles.tsx]: Why was this removed?

Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/TextField/TextField.styles.tsx]: I moved it town to 206 inside of `!disabled` otherwise disabled controls get a hover border 
textfield
